src/coreclr/pal/inc/pal.h

```diff
@@ -2619,6 +2763,19 @@ PALIMPORT BOOL PALAPI PAL_VirtualUnwindOutOfProc(CONTEXT *context, KNONVOLATILE_
 
 PALIMPORT BOOL PALAPI PAL_GetUnwindInfoSize(SIZE_T baseAddress, ULONG64 ehFrameHdrAddr, UnwindReadMemoryCallback readMemoryCallback, PULONG64 ehFrameStart, PULONG64 ehFrameSize);
 
+// Machine architectures
+// See current registered ELF machine architectures at:
+//    https://github.com/torvalds/linux/blob/master/include/uapi/linux/elf-em.h
+
+#define EM_386          3
+#define EM_PPC64        21
+#define EM_S390         22
+#define EM_ARM          40
+#define EM_X86_64       62
+#define EM_AARCH64      183
+#define EM_RISCV        243
+#define EM_LOONGARCH    258
+
```

Comment:
These are the only few values we are using from llvm/ELF.h, and they are defined in ELF spec. We don't need to sync the entire header (which is already diverged too much) with llvm upstream for these.